MoviesApp/MoviesApp/FirstScreen/Presenter/FilmsViewProtocol.swift

```diff
@@ -0,0 +1,10 @@
+// FilmsViewProtocol.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Films View Protocol
```

Comment:
```suggestion
/// Films View Protocol
```
Коменты или все на русском или все на англ